networking/src/ModelHelper.ts

```diff
@@ -109,14 +109,19 @@ export abstract class ModelHelper {
 
         return Math.round(totalNoChosenAnswer / gameSession.teams.length * 100)
     }
-
-    static calculateBasicModeScoreForQuestion(gameSession: IGameSession, question: IQuestion, team: ITeam) {
```

Comment:
scoring gets updated to read from the new response object from the question, instead of the choices array. This is so we don't have to run the same equality checks on the answers and just rely on the object we've already created when receiving the answers.